lib/ansible/modules/replace.py

```diff
@@ -110,7 +110,8 @@
 '''
 
 EXAMPLES = r'''
-- name: Before Ansible 2.3, option 'dest', 'destfile' or 'name' was used instead of 'path'
+# NOTE: Before Ansible 2.3, option 'dest', 'destfile' or 'name' was used instead of 'path'
+- name: Replace old hostname with new hostname
```

Comment:
The task name is an appropriate place to put comments. I'm not sure the example needs to describe 2.3 at all though (or at least not in such detail) since the example works in 2.4+. 

```suggestion
- name: Replace old hostname with new hostname (requires Ansible >= 2.4)
```

Reply:
the `# `note will be removed when building/display docs and the example is a YAML document, This why we actually went the opposite direction and removed most `#` comments in favor of putting the information in `name`